File: test_contest/solutions/power.c
/**
 * The Power of Two is a Curious Thing
 * Author: Tyler Woodhull
 */

// Include the necessary header
#include <stdio.h>

int main() {
	// Create variables needed to solve the problem
	int i, caseCount, n;

	// Scan the number of test cases
	scanf("%d", &caseCount);

	// Iterate through all test cases
	for (i = 0; i < caseCount; i++) {
		// Scan the value for n
		scanf("%d", &n);
		// Print (2 ^ n) - 1 using a bitshift
		printf("%d\n", (1 << n) - 1);
	}

	return 0;
}

File: test_contest/solutions/power.java

import java.util.Scanner;

public class power {

	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);
		
		int cases = scan.nextInt();
		for(int i = 0;i<cases;i++){
			int pow = scan.nextInt();
			System.out.println((int)Math.pow(2, pow) - 1);
		}

	}

}
